Jenkinsfile

```diff
@@ -155,10 +156,15 @@ pipeline {
         buildApplication()
       }
     }
+
+    // Only run chromatic tests on latest
```

Comment:
Is this comment accurate? The changes below relate to the *Test Development* and *Test Production* stages.

Reply:
Oops, left the comment in the wrong place